web-ui/src/main/resources/catalog/views/default/templates/footer.html

```diff
@@ -1,4 +1,4 @@
-<ul class="nav navbar-nav">
+<ul class="nav navbar-nav" ng-show="getApplicationInfoVisible()">
```

Comment:
```suggestion
<ul class="nav navbar-nav" ng-show="::getApplicationInfoVisible()">
```
should work fine and avoid multiple checks.